packages/ui/src/themes/theme.styled.ts

```diff
@@ -69,6 +69,7 @@ export const theme = {
       toggleActive: colors.background.black,
       toggle: colors.background.toggleOff,
       list: colors.background.list,
+      infoBackground: colors.background.infoBackground,
```

Comment:
Since the object is already named `background`, just `info` is suffisient
```suggestion
      info: colors.background.info,
```

Reply:
done